src/main/java/com/github/games647/flexiblelogin/commands/admin/AccountsCommand.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2018 Toranktto.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.github.games647.flexiblelogin.commands.admin;
+
+import com.github.games647.flexiblelogin.FlexibleLogin;
+import com.github.games647.flexiblelogin.commands.AbstractCommand;
+import com.github.games647.flexiblelogin.config.Settings;
+import com.github.games647.flexiblelogin.storage.Account;
+import com.github.games647.flexiblelogin.validation.IPv4Predicate;
+import com.github.games647.flexiblelogin.validation.NamePredicate;
+import com.github.games647.flexiblelogin.validation.UUIDPredicate;
+import com.google.inject.Inject;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import static org.spongepowered.api.command.args.GenericArguments.onlyOne;
+import static org.spongepowered.api.command.args.GenericArguments.string;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.scheduler.Task;
+import static org.spongepowered.api.text.Text.of;
+
+public class AccountsCommand extends AbstractCommand {
+
+    @Inject
+    private UUIDPredicate uuidPredicate;
+
+    @Inject
+    private NamePredicate namePredicate;
+
+    @Inject
+    private IPv4Predicate ipv4Predicate;
```

Comment:
Please use the [guava equivalent instead](https://google.github.io/guava/releases/snapshot-jre/api/docs/com/google/common/net/InetAddresses.html#isInetAddress-java.lang.String-). It's well tested using some unit tests.